package.json

```diff
@@ -11,7 +11,7 @@
     "github-contributions-canvas": "^0.6.0",
     "lodash": "^4.17.19",
     "next": "^13.1.1",
-    "node-fetch": "^2.6.0",
+    "node-fetch": "^2.6.7",
```

Comment:
Since Next.js ships with fetch, you shouldn’t need node-fetch installed. Also, it’s a global since Node.js 18

Reply:
@styfle done